foosball/scoreboard/main.fbp

```diff
@@ -32,6 +32,9 @@ Score(ScoreLed)
 ScoreSwitcher(switcher/int:keep_state=true)
 RedWin(RedWinLed)
 YellowWin(YellowWinLed)
+P1Sound(piezo-speaker:tune="aabbccdd")
+P2Sound(piezo-speaker:tune="aabbccdd")
+
```

Comment:
are you going to play the same sound if any player wins?
I believe it would be better to use different ones.
But if you really want to play the same sound there is no reason to have two different nodes, use a single node and connect both ports to it:

ScoreBoard P1_WIN -> ENABLED PSound
ScoreBoard P2_WIN -> ENABLED PSound